dev/io.openliberty.org.jboss.resteasy.common/resources/io/openliberty/org/jboss/resteasy/common/nls/RESTfulWSServer.nlsprops

```diff
@@ -76,8 +66,30 @@ INVALID_PROVIDER_CWWKW1305W=CWWKW1305W: The server is ignoring the {0} provider
 INVALID_PROVIDER_CWWKW1305W.explanation=The specified provider class likely does not contain a valid public constructor or does not implement a Jakarta REST provider interface. The requirements for a valid provider implementation are documented in section 4 of the Jakarta RESTful Web Services specification.
 INVALID_PROVIDER_CWWKW1305W.useraction=Review the application code to ensure that the specified provider properly implements the provider interface and contains a valid public constructor per the specification requirements.
 
+INVALID_LONG_PROPERTY_CWWKW1306W=CWWKW1306W: The {0} client property must be a long integer, but was configured incorrectly with {1}. This setting will be ignored.
+INVALID_LONG_PROPERTY_CWWKW1306W.explanation=Client properties, such as timeout settings, must be configured as a positive long integer or a string that can be converted to a long integer. This message indicates that a client property was configured with an invalid value. This property will be ignored.
+INVALID_LONG_PROPERTY_CWWKW1306W.useraction=Review the application code and server configuration to determine where this property was set incorrectly and either remove the property or change it to a valid long integer value.
+
+# Same message as in /com.ibm.ws.jaxrs.2.0.common/resources/com/ibm/ws/jaxrs20/common/internal/resources/JaxRsCommonMessages.nlsprops
+
+warn.invalid.authorization.token.type=CWWKW0061W: The  {0} authorization token type specified in the server configuration is invalid and will be ignored.
+warn.invalid.authorization.token.type.explanation=The authorization token type specified in the server configuration is invalid.
+warn.invalid.authorization.token.type.useraction=Specify a valid authorization token type.
+
+# Same messages as in /com.ibm.ws.jaxrs.2.0.client/resources/com/ibm/ws/jaxrs20/client/internal/resources/JAXRSClientMessages.nlsprops
+
+failed_to_extract_saml_token_from_subject=CWWKW0705W: An exception occurred while attempting to use the SAML PropagationHelper API. The exception was: [{0}]
+failed_to_extract_saml_token_from_subject.explanation=An exception occurred while invoking the PropagationHelper API to retrieve the SAML token from the subject.
+failed_to_extract_saml_token_from_subject.useraction=Ensure that the server.xml has the samlWeb-2.0 feature specified. Also, check the server logs for additional information regarding the exception. 
```

Comment:
```suggestion
failed_to_extract_saml_token_from_subject.useraction=Ensure that the samlWeb-2.0 feature is specified in the server.xml file. Also, check the server logs for additional information regarding the exception. 
```

Reply:
I cannot change this message since it already exists and is copied from com.ibm.ws.jaxrs.2.0.client/resources/com/ibm/ws/jaxrs20/client/internal/resources/JAXRSClientMessages.nlsprops